### ISSUE TITLE
Untranslated WCAG 1.1 page in fr

### ISSUE THREAD

**Issue by SphinxKnight** [mdn/translated-content#7417]
/fr/docs/Web/Accessibility/Understanding_WCAG/Perceivable#guideline_1.1_—_providing_text_alternatives_for_non-text_content is completely in English

**Comment by SphinxKnight** [comment]
This page has a child and should be translated instead of deleted
